.github/workflows/puppeteer.yml

```diff
@@ -24,7 +24,6 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@5e21ff4d9bc1a8cf6de233a3057d20ec6b3fb69d # v3.8.1
         with:
-          cache: npm
```

Comment:
It's very easy for this to be reintroduced in a future refactoring that tries to make all actions consistent (with cache). A tombstone comment here would have been helpful.